### ISSUE TITLE
SecurityHub data validation fails

### ISSUE THREAD

**Issue by karthik12112** [prowler-cloud/prowler#778]
Resources block Type cant be empty when sending data to SecurityHub
```
"ErrorMessage": "Finding does not adhere to Amazon Finding Format. data.Resources[0].Type should NOT be shorter than 1 characters."
```
Sample output

```
{
         "SchemaVersion": "2018-10-08",
         "Id": "*****",
         "ProductArn": "arn:aws:securityhub:us-west-2::product/prowler/prowler",
         "RecordState": "ACTIVE",
         "ProductFields": {
           "ProviderName": "Prowler",
           "ProviderVersion": "2.4.0-07042021"
         },
         "GeneratorId": "prowler-check12",
         "AwsAccountId": "******",
         "Types": [
           "ens-op.acc.5.aws.iam.1"
         ],
         "FirstObservedAt": "2021-04-15T07:15:49Z",
         "UpdatedAt": "2021-04-15T07:15:49Z",
         "CreatedAt": "2021-04-15T07:15:49Z",
         "Severity": {
           "Label": "HIGH"
         },
         "Title": "[check12] Ensure multi-factor authentication (MFA) is enabled for all IAM users that have a console password (Scored)",
         "Description": "User **** has Password enabled but MFA disabled",
         "Resources": [
           {
             "Type": "",
             "Id": "AWS::::Account:****",
             "Partition": "aws",
             "Region": "us-west-2"
           }
         ],
         "Compliance": {
           "Status": "FAILED",
           "RelatedRequirements": [
             "ens-op.acc.5.aws.iam.1"
           ]
         }
       }

**Comment by toniblyx** [comment]
Thanks, that bug has been fixed in branch 2.4.1.

**Comment by toniblyx** [comment]
Now in master and release 2.4.1
